{
	"schema_version": "1.6.0",
	"id": "BIT-gitlab-2024-6595",
	"modified": "2024-11-05T07:36:31.248Z",
	"published": "2024-07-19T07:19:00.729Z",
	"aliases": [
		"CVE-2024-6595"
	],
	"details": "An issue was discovered in GitLab CE/EE affecting all versions starting from 11.8 prior to 16.11.6, starting from 17.0 prior to 17.0.4, and starting from 17.1 prior to 17.1.2 where it was possible to upload an NPM package with conflicting package data.",
	"affected": [
		{
			"package": {
				"ecosystem": "Bitnami",
				"name": "gitlab",
				"purl": "pkg:bitnami/gitlab"
			},
			"severity": [
				{
					"type": "CVSS_V3",
					"score": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:L/A:N"
				}
			],
			"ranges": [
				{
					"type": "SEMVER",
					"events": [
						{
							"introduced": "11.8.0"
						},
						{
							"fixed": "16.11.6"
						},
						{
							"introduced": "17.0.0"
						},
						{
							"fixed": "17.0.4"
						},
						{
							"introduced": "17.1.0"
						},
						{
							"fixed": "17.1.2"
						}
					]
				}
			],
			"database_specific": {
				"source": "https://github.com/bitnami/vulndb/tree/main/data/gitlab/BIT-gitlab-2024-6595.json"
			}
		}
	],
	"references": [
		{
			"type": "WEB",
			"url": "https://blog.vlt.sh/blog/the-massive-hole-in-the-npm-ecosystem"
		},
		{
			"type": "WEB",
			"url": "https://gitlab.com/gitlab-org/gitlab/-/issues/417975"
		}
	],
	"database_specific": {
		"cpes": [
			"cpe:2.3:a:gitlab:gitlab:*:*:*:*:*:*:*:*",
			"cpe:2.3:a:gitlab:gitlab:*:*:*:*:enterprise:*:*:*"
		],
		"severity": "Medium"
	}
}
